client/webpack.config.js

```diff
@@ -221,7 +221,8 @@ module.exports = function exports(env, argv) {
         {
           test: /\.(js|ts)x?$/,
           loader: 'ts-loader',
-          exclude: /node_modules/,
+          // ensure stimulus (ES6 npm module) is transpiled to ES5 - required until whole code base is moved to ES6
+          exclude: /[\\/]node_modules[\\/](?!(@hotwired\/stimulus)[\\/])/,
```

Comment:
@thibaudcolas here is the node modules change needed for Stimulus to work with our existing tsconfig / webpack compilation target of ES5. 

https://github.com/wagtail/rfcs/pull/78/files#r980926580